webapp-django/crashstats/topcrashers/views.py

```diff
@@ -104,8 +105,19 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
-            # Number of startup crashes.
-            hit['startup_percent'] = 0
+            # Number of crashes happening during startup. This is defined by
+            # the software, as opposed to the next methid which relies on
```

Comment:
Would ``client`` be a better term here? I want to say it is defined by Firefox/Breakpad but using a generic term.